variants/STM32G4xx/G473C(B-C-E)U_G474C(B-C-E)U_G483CEU_G484CEU/generic_clock.c

```diff
@@ -23,8 +23,65 @@
   */
 WEAK void SystemClock_Config(void)
 {
-  /* SystemClock_Config can be generated by STM32CubeMX */
-#warning "SystemClock_Config() is empty. Default clock at reset is used."
+  RCC_OscInitTypeDef RCC_OscInitStruct = {0};
+  RCC_ClkInitTypeDef RCC_ClkInitStruct = {0};
+#ifdef USBCON
+  RCC_CRSInitTypeDef pInit = {0};
+  RCC_PeriphCLKInitTypeDef PeriphClkInit = {0};
+#endif
```

Comment:
Remove `0` which raised warnings.
```suggestion
  RCC_OscInitTypeDef RCC_OscInitStruct = {};
  RCC_ClkInitTypeDef RCC_ClkInitStruct = {};
#ifdef USBCON
  RCC_CRSInitTypeDef pInit = {};
  RCC_PeriphCLKInitTypeDef PeriphClkInit = {};
#endif
```